addons/web/static/src/views/fields/many2one/many2one_field.xml

```diff
@@ -52,6 +52,7 @@
                         aria-label="Internal link"
                         data-tooltip="Internal link"
                         t-on-click="onExternalBtnClick"
+                        t-ref="middleclick"
```

Comment:
Why not just modify the `<button` in a link and add a href ?

Reply:
I wanted to use a common implementation under a hook, instead of implementing it differently at multiple places. For simple buttons, yes we could use href directly, but not for clickable elements such as list rows/kanban card, it is not possible at all.

Also, it's not the most visible for this button, but I added some opacity change when pressing ctrl key, to visualize elements that do support "open in new tab"